### ISSUE TITLE
Issue with using QueryParser to parse TREC topics

### ISSUE THREAD

**Issue by lintool** [lintool/Anserini#26]
We're currently using QueryParser to parse TREC topics, which means that symbols in the topics like parentheses and quotes get interpreted as query operators... this isn't the desired behavior.

**Comment by lintool** [comment]
Here are some example of differences:

`Oprah Winfrey half-sister` gets interpreted as Lucene query:

> text:oprah text:winfrei (text:half text:sister)

And `release of "The Rite"` as

>text:releas text:of text:"the rite"

`Super Bowl, seats`

> text:super (text:bowl text:) text:seat

**Comment by lintool** [comment]
Interestingly:

```
TREC 2011
QueryParser   MAP = 0.3577  P30 = 0.4000
BagOfWords   MAP = 0.3532  P30 = 0.3993

TREC 2012
QueryParser   MAP = 0.2091  P30 = 0.3311
BagOfWords   MAP = 0.2090  P30 = 0.3311
```

So the weird QueryParser interpretation of query operators helps a tiny bit for TREC 2011!

**Comment by lintool** [comment]
Bottom line: better to do it "right" rather than depend on something completely idiosyncratic.

**Comment by lintool** [comment]
Resolved in commit 36cafb706d4cf6661b560e78b33bf2ec22b29dfa
